components/03-modules/header/_header.scss

```diff
@@ -13,7 +13,7 @@ $header__icon-color            : $gray-dark !default;
         display: flex;
         flex-flow: row wrap;
         @include mq($screen-xl) {
-            margin: 2px 0;
+            margin: 4px 0;
```

Comment:
shouldn't we move it to component variable?

Reply:
done.